pygsheets/cell.py

```diff
@@ -171,6 +173,21 @@ def set_text_format(self, attribute, value):
         self.update()
         return self
 
+    def set_text_rotation(self, attribute="angle", value=0):
+        """
+        set the text rotation
+        :param attribute: "angle"
+        :param value: corresponding value for the attribute
+        :return: :class: Cell
+        """
+        if attribute != "angle":
```

Comment:
Yes vertical is also a possible param but I was unable to set it. When I try to set vertical I have this following error: oneof field 'type' is already set. Cannot set 'vertical'